### PR TITLE
feat: add number of employees for each department

### PR DESCRIPTION
**Pull request by MaskyS**
1. Added 4 missing departments: 
	- 	[Law Library](https://codelibrary.amlegal.com/codes/san_francisco/latest/sf_charter/0-0-0-575)* 
	- [Adult Probation Department](https://codelibrary.amlegal.com/codes/san_francisco/latest/sf_admin/0-0-0-3408) 
	- [Animal Care and Control Department](https://codelibrary.amlegal.com/codes/san_francisco/latest/sf_health/0-0-0-112)
	- [California Academy of Sciences](https://codelibrary.amlegal.com/codes/san_francisco/latest/sf_charter/0-0-0-1369)* 

\* this is missing in `sfg-graph` too

2. Added number of employees, manually copied from "Department Totals"  rows in [SF Salary Ordinance FY25-26](https://www.sf.gov/sites/default/files/2024-07/ASO%20FY2024-25%20%26%20FY2025-26%20-%20FINAL%207.31.24.pdf)
**Notes**: Some departments actually have their no of employees under other departments, as a division. This means if we ever tally up the employee number across departments, it'll be inaccurate. Maybe should add a new `parent_department` field?
  - `Disability and Aging Services` under `HSA Human Services`  
Screenshot 2024-10-18 at 5 53 47 PM
  - `Entertainment Commission` under `City Admin` aka `ADM General Services Agency`
  Screenshot 2024-10-18 at 5 52 37 PM

  - `Film Commission` under `ECN Economic & Workforce Development`
Screenshot 2024-10-18 at 5 51 16 PM
  - `Office of Small Business` under `ECN Economic & Workforce Development`
Screenshot 2024-10-18 at 5 50 33 PM
- `Animal Care and Control` under `City Admin` aka `ADM General Services Agency`
Screenshot 2024-10-18 at 8 01 24 PM

### CODE REVIEW

File: sf_gov_departments.csv [hunk 1]
```diff
@@ -1,53 +1,56 @@
...
-"San Francisco Public Utilities Commission, General Manager","Public Utilities Commission","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Sheriff","Sheriff's Department","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Sheriff's Department Office of Inspector General, Inspector General","Sheriff's Department Office of Inspector General","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Small Business, Director","Office of Small Business","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Technology, Executive Director","Department of Technology","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"Treasurer","Office of the Treasurer & Tax Collector","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
-"War Memorial and Performing Arts Center, Managing Director","War Memorial","Campaign & Governmental Conduct Code Sec. 3.1-103(b)(1)"
\ No newline at end of file
+﻿"Department Heads","Departments","Departments, Legal Source","2024-2025 Full-Time Employees (FTE)","2025-2026 Full-Time Employees (FTE)"
+"Academy of Sciences, Executive Director","California Academy of Sciences","Charter Sec. 16.105","13.34","13.34"
```
Comment (MaskyS) [inline sf_gov_departments.csv:2]: Their org chart from https://www.sf.gov/sites/default/files/2024-05/CSF_Proposed_Budget_Book_June_2024_r8.pdf has Board of Trustees at the top, but I thought Executive Director made more sense...

Screenshot 2024-10-18 at 5 42 37 PM

Comment (m-atoms) [reply sf_gov_departments.csv:2]: Yeah Board of Trustees would be "commission" level and Executive Director would be department head level

**Comment by m-atoms** [comment]
- This addresses #8 and #10... still weird that it's missing from the municipal code
- Law Library definitely needs to be added but then we'll also need to add the Board of Trustees at the commission level
- Same for Academy of Sciences

**Comment by m-atoms** [comment]
looks good, nice work

**Pull request merged**